LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
The [official plugins](https://github.com/getsentry/sentry-plugins) use this license and I took a lot of code snippets from there, so better avoid any copyright problem.
Anyway, the Apache 2.0 license is also very common and is basically like MIT and BSD-3-clause, but with additional protection against patent lawsuits.